### ISSUE TITLE
[FP]: CVE-2022-31548 querydsl-core-4.4.0.jar, querydsl-apt.4.4.0.jar, querydsl-jpa-4.4.0.jar, querydsl-codegen-4.4.0.jar 

### ISSUE THREAD

**Issue by Janpopan** [dependency-check/DependencyCheck#4690]
### Package URl

pkg:maven/com.querydsl/querydsl-core@4.4.0, 
pkg:maven/com.querydsl/querydsl-apt@4.4.0, 
pkg:maven/com.querydsl/querydsl-jpa@4.4.0,
pkg:maven/com.querydsl/querydsl-codegen@4.4.0

### CPE

cpe:2.3:a:homepage_project:homepage:4.4.0:*:*:*:*:*:*:* 

### CVE

CVE-2022-31548

### ODC Integration

{"label"=>"Maven Plugin"}

### ODC Version

7.1.1

### Description

Looks the same "problem" like CVE-2022-31569
#4671 

it is the same like
#4677

**Comment by hohwille** [comment]
I am facing the same problem with latest available release 7.1.1:
```
[INFO] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '8,0':

querydsl-core-5.0.0.jar: CVE-2022-31548(9.3)
```

So a PR already exists that seems to fix it. Thanks @marcelstoer  already. Can I assume that this is going to be merged and a release 7.x will come out in several weeks that will fix this? Or is that a wrong expecation?

Just for the record. IMHO and just from my personal PoV the entire CVE matching approach seems to be flawed and maintaining manual lists of CVE exclusions is more of a workaround than a real fix.

**Comment by marcelstoer** [comment]
> Can I assume that this is going to be merged 

@jeremylong's latest comment at https://github.com/jeremylong/DependencyCheck/pull/4688#issuecomment-1202271891 is a confirmation to that effect. 

> the entire CVE matching approach seems to be flawed and maintaining manual lists of CVE exclusions is more of a workaround than a real fix.

👍 to that.
